src/core/wee-network.c

```diff
@@ -311,6 +312,9 @@ network_resolve (const char *hostname, char *ip, int *version)
 
     res = NULL;
 
+    if (!res_init())
```

Comment:
According to `man res_init`, the function returns 0 on success, -1 if an error occurs.
So I think the test should be: `if (res_init() != 0)`
Same problem in other tests in this commit.